docs: supported hardwares (#230)
cloud-gouv/securix: docs/manual/src/user/hardware.md

<!--
SPDX-FileCopyrightText: 2026 Antoine Eiche <antoine.eiche@lewocorp.eu>

SPDX-License-Identifier: CC-BY-SA-4.0
-->

# Hardware profiles

The SecurixOS team maintains a set of hardwares profiles in the `hardware` directory. These hardware profiles have been manually tested and are used in real-world environments.

> [!WARNING]

> > Currently, the team doesn't accept hardware contributions from the community because they could not be validated by the project team.

Reviewer: ```suggestion
> Currently, the team doesn't accept hardware contributions from the community we cannot test and validate continuously.
```

However, if your hardware is not supported, you could still use it in SecurixOS by creating and importing your own module in your configuration. A SecurixOS hardware module file `your-hardware.nix` looks like:

```nix
{ pkgs, lib, config, ...}:
{

>   options.securix.self.machine.hardwareSKU = mkOption {

Reviewer: Put a reference to type merging to explain it for those who may be confused about this

Author: Done.
